### ISSUE TITLE
#-798964761------------- Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#828]
<strong>Source:</strong>`<div class="sc-kgflAQ bAlVxb">Product Owner, Survey123</div>`
<strong>Element Location: </strong>div:nth-child(2) > .gWxTCi.StyledListItem-sc-1sb8owe-1.feUFJp > .StyledListTextContainer-sc-1sb8owe-5.bgEsee > .StyledListSubtitle-sc-1sb8owe-4.jhpUkl.sc-dmRaPn > .sc-kgflAQ.bAlVxb:nth-child(2)
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix any of the following:
  Element has insufficient color contrast of 3.03 (foreground color: #949494, background color: #ffffff, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1
Tags: cat.color,wcag2aa,wcag143,ACT
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
